packages/react/src/runtime/disable-cache.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns `true` when caching should be disabled, else `false`.
+ *
+ * Any code within this check will be removed in the output bundles:
+ *
+ * ```js
+ * if (isCacheDisabled()) {
+ *   // This code will be removed from the output bundles
+ * }
+ * ```
+ */
+export const isCacheDisabled = (): boolean => {
```

Comment:
I would name the file `is-cache-disabled` or `cache`

Reply:
I'll go with `cache` - I was so torn on which name to pick 😖 haha

Reply:
It might be good to mark this explicitly as used only in tests? Both the file name and function being called something obvious like `INTERNAL_TESTING_ONLY_isStyleCacheDisabled()`. Love the jsdoc mentioned code is removed during bundling! It might also be good to add extra context why it was needed to be used?

Is what we're doing only needed to test the sheet insertion logic? Could consumers need this capability as well (is there a bigger story we're missing?)